### PR TITLE
Cover/cmds/core/find

### PR DESCRIPTION
**Pull request by ChriMarMe**

### CODE REVIEW

**Comment by ChriMarMe** [comment]
```
09:56:06 Build environment: GOARCH=amd64 GOOS=linux GOROOT=/usr/local/go GOPATH=/go CGO_ENABLED=0
09:56:22 Build error: error building: error building go package in "/go/src/github.com/u-root/u-root/bb": # github.com/u-root/u-root/cmds/core/find/.bb
../cmds/core/find/.bb/find.go:43:23: undefined: fs
../cmds/core/find/.bb/find.go:67:15: assignment mismatch: 2 variables but 1 value
, exit status 2

Exited with code exit status 1
```

I have no idea what is going wrong.

File: cmds/core/find/find.go [hunk 1]
```diff
@@ -17,84 +17,97 @@ import (
...
+func init() {
+	flag.IntVar(&fargs.perm, "mode", -1, "Permissions")
+	flag.StringVar(&fargs.filetype, "type", "", "File type")
+	flag.StringVar(&fargs.name, "name", "", "glob for name")
+	flag.BoolVar(&fargs.long, "l", false, "long listing")
+	flag.BoolVar(&fargs.debug, "d", false, "Enable debugging in the find package")
+}
+
+func runFind(out io.Writer, errOut io.Writer, fargs flags, arg []string) error {
+	if len(arg) != 1 {
```
Comment (rminnich) [inline cmds/core/find/find.go:59]: well guess what :-)
0 args is legal. Should we do that?

**Review by rminnich — APPROVED** [review]

File: cmds/core/find/find.go (hunk 1 above)
Comment (ChriMarMe) [reply cmds/core/find/find.go:59]: I'm not sure, but you do realize the CI failed?

**Comment by ChriMarMe** [comment]
I really want to figure out how to please the CI here.

**Pull request closed without merging**